airgun/entities/host_new.py

```diff
@@ -292,6 +292,15 @@ def apply_package_action(self, entity_name, package_name, action):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def get_errata_ids(self, entity_name):
```

Comment:
This isn't specifically getting the errata ids, this is just reading the errata table. Either change the functionality, or change the name.

Reply:
@sambible I renamed to `get_errata_table()`, and added optional params to filter/search, 
since there is no method yet (in new host entity), that can filter by multiple params and search at the same time.
Let me know what you think!

Reply:
nice work